### DISCUSSION TITLE
Sidebar editor - Add a DSL for sidebar configuration

### DISCUSSION THREAD

**Discussion by calebuharrison** [avo-hq/avo#399] [Feature requests]
Currently, if you want to customize the sidebar (aside from hiding individual resources), you have to eject the sidebar partial and completely rewrite it. It would be nice to have a resource-like DSL that would allow for custom ordering, authorization, etc.

**Comment by adrianthedev** [comment]
We knew about this issue since we added the custom tools feature. We'll start brainstorming this feature.

Have you worked with something similar in the past? Do you have something in mind?

**Comment by adrianthedev** [comment]
Fixed in https://github.com/avo-hq/avo/pull/829
